### PR TITLE
Slack reply broadcast

### PR DESCRIPTION
**Pull request by MWEJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added support for the reply_broadcast parameter as listed on https://api.slack.com/methods/chat.postMessage

## Screenshots
Emulates the use of the following checkbox in the Slack client
image

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Slack/Integrations/SlackV3/SlackV3.py [hunk 1]
```diff
@@ -1730,6 +1730,7 @@ def slack_send():
     thread_id = args.get('threadID', '')
     severity = args.get('severity')  # From server
     blocks = args.get('blocks')
+    reply_broadcast = args.get('reply_broadcast')
```
Comment (AradCarmi) [inline Packs/Slack/Integrations/SlackV3/SlackV3.py:1733]: The cast is required as the input type is a string.
```suggestion
    reply_broadcast = argToBoolean(args.get('reply_broadcast'))
```

File: Packs/Slack/ReleaseNotes/3_0_3.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Slack v3
+- Added support for the reply_broadcast parameter to the send-notification command
```
Comment (AradCarmi) [inline Packs/Slack/ReleaseNotes/3_0_3.md:4]:
```suggestion
- Added support for the *reply_broadcast* parameter to the **send-notification** command.
```

**Review by AradCarmi — CHANGES_REQUESTED** [review]
Hey @MWEJ,
Excellent job! 🚀
See my suggestions below and let me know when you are available for a short demo. Thanks again for the contribution! 💯

**Comment by MWEJ** [comment]
> Hey @MWEJ, Excellent job! 🚀 See my suggestions below and let me know when you are available for a short demo. Thanks again for the contribution! 💯

I'm available most of the time today (working US PDT hours), but it's a rather simple functionality that doesn't need much explanation

**Comment by AradCarmi** [comment]
I cannot meet today if you can prepare a short video explaining the changes and display the broadcast it will be enough.

**Comment by AradCarmi** [comment]
I'm summing up the notes that need to be resolved in order to merge our PR:
1. Please pull from origin/master.
2. Please bump the current release note version (to 3_0_4), by running:
`demisto-sdk update-release-notes -i Packs/Slack -u revision`
3. Please fix the` Run Unit Testing And Lint `errors regarding some broken Unit-Tests caused by the change.
Thanks again,
Arad

**Comment by AradCarmi** [comment]
Hey @MWEJ, 
Any update regarding my suggestions?
Thanks!

**Comment by AradCarmi** [comment]
Hey @MWEJ, 
I tried to reach out and didn't get a response.
Please feel free to re-open the PR.
Thanks!

**Pull request closed without merging**